snorkel/augmentation/tf.py

```diff
@@ -10,13 +9,7 @@
 """
 
 
-class BaseTransformationFunction(BaseMapper):
-    """Base class for `TransformationFunction` and `transformation_function`.
-
-    See `snorkel.map.core.BaseMapper` for details.
-    """
-
-    pass
+BaseTransformationFunction = BaseMapper
```

Comment:
The inheritance above is a pattern we've used elsewhere to "alias" classes—we should be consistent?

Reply:
It's also unclear where this is used, given there's a base class for TFs, `TransformationFunction`, right below... why do we need both?  (A) there should at least be a comment here explaining, but more importantly, this seems *very* heavyweight if we really do need both...

Reply:
This is only used for type annotations, and mypy balks if you subclass like we do above. Having this alias IMO makes the TF files more legible.

Reply:
Okay well at least comment explaining this!